sandbox

```diff
@@ -169,6 +169,25 @@ sandbox () {
     docker-compose -f "$SANDBOX_DIR/docker-compose.yml" "$@"
   }
 
+  dc_no_cache () {
+    docker-compose build --no-cache
```

Comment:
Use `dc` instead of `docker-compose`, as `dc` specifies correctly the `docker-compose.yml` file